adafruit_max31856.py

```diff
@@ -279,7 +288,9 @@ def temperature_thresholds(self, val):
         self._write_u8(_MAX31856_LTLFTL_REG, int_low)
 
     @property
-    def reference_temperature_thresholds(self):  # pylint: disable=invalid-name
+    def reference_temperature_thresholds(  # pylint: disable=invalid-name,
+        self,
+    ) -> Tuple[float, float]:  # pylint: disable=invalid-name
```

Comment:
Multiline `pylint` is a pain.  You can get rid of on of these `# pylint: disable=invalid-name` comments, and I think it's THIS one.

Reply:
I think originally they were correct and black broke up the line after the annotations were added....I missed the dup when I added the corrections....

Reply:
Yeah, black breaks moves it to the bottom line, but then pylint just gets confused again.  It's one of the few bad pairings of the two I commonly run into myself.